### PR TITLE
Status added on landing screen

### LINKED ISSUE #71: Provide some kind of form status summary on landing screen
Currently, there's no way to tell which forms need to be reviewed, which have been received, etc. A small summary either in the summary text or on the right side would help a user figure out which forms are worth clicking on.

### PR DESCRIPTION
**Pull request by lakshyagupta21**
Closes #71 

Added `reviewed `and `unreviewed `form status on landing page.

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
@lakshyagupta21 The count of the `reviewed` and `unreviewed` is not getting updated on the main screen but it is getting properly updated in the statistics screen

File: share_app/src/main/java/org/odk/share/activities/MainActivity.java [hunk 1]
```diff
@@ -58,13 +61,15 @@ protected void onCreate(Bundle savedInstanceState) {
         LinearLayoutManager llm = new LinearLayoutManager(this);
         llm.setOrientation(LinearLayoutManager.VERTICAL);
         recyclerView.setLayoutManager(llm);
+        setupAdapter();
+        getSupportLoaderManager().initLoader(FORM_LOADER, null, this);
     }
 
     @Override
     protected void onResume() {
+        updateAdapter();
```
Comment (shobhitagarwal1612) [inline share_app/src/main/java/org/odk/share/activities/MainActivity.java]: Is there a particular reason as to why `super.onResume()` is not called the first statement?

File: share_app/src/main/res/layout/form_item_list.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="8dp">
+
+    <RelativeLayout
```
Comment (shobhitagarwal1612) [inline share_app/src/main/res/layout/form_item_list.xml]: The layout can also be created without using the extra two `RelativeLayouts`

**Comment by lakshyagupta21** [comment]
> The count of the reviewed and unreviewed is not getting updated on the main screen but it is getting properly updated in the statistics screen

There are some issues with the cursor loader init and restart I tried to resolve it in #82 but it leads to the crash, I'll resolve this in another PR.

**Comment by shobhitagarwal1612** [comment]
@lakshyagupta21 Please open another issue to keep a log of the problem and this PR looks good to be merged

**Comment by lakshyagupta21** [comment]
@lognaturel Please try this PR again. If you still face the issue then please let me know the procedure to reproduce it.

**Comment by shobhitagarwal1612** [comment]
@lakshyagupta21 Are you still interested in working on this issue?

**Comment by lakshyagupta21** [comment]
@shobhitagarwal1612 Sorry for keeping this PR in working state for so long. I'll try to fix the issue raised in the above comments by tomorrow.

**Comment by shobhitagarwal1612** [comment]
No rush. I wanted to just confirm if you want to finish it since you've already implemented it. Otherwise I'd be happy to continue from here

**Comment by lakshyagupta21** [comment]
If you want to work on it, go ahead I'll take some other issue.

**Comment by lakshyagupta21** [comment]
@shobhitagarwal1612 Did you get a chance to review and test this?

**Comment by shobhitagarwal1612** [comment]
I did test it then but forgot to update the status here. Will test it again today

**Pull request merged**